docs/api.md

```diff
@@ -709,6 +710,23 @@ curl -H "GOVUK-Account-Session={{account_session_header}}" \
 }
 ```
 
+### `POST /api/oidc_events/backchannel_logout`
+
+A public API endpoint that Implements OpenID Connect Back-Channel Logout 1.0 - draft 07.
+
+#### Query parameters
+- `logout_token`
+  - A signed JWT
+
+#### Response codes
+
+- 400 if the Logout Token JWT is invalid or signature cannot be verified
+- 200 otherwise
+
+#### Note
+
+This endpoint is intended for calls to be made exclusively from the Digital Identity OP to the GOV.UK relaying party. As such it will not appear in GDS API Adaptors as it is not intended to be called from another Ruby compatible
```

Comment:
I'm not sure what you mean by "not intended to be called from another Ruby compatible"